Description: This patch fixes some more spelling errors detected by the Debian
 tool lintian.
Author: Paul Gevers <elbrus@debian.org>
Bug: http://mantis.freepascal.org/view.php?id=31112

Index: fpc/install/man/man1/fpc.1
===================================================================
--- fpc.orig/install/man/man1/fpc.1
+++ fpc/install/man/man1/fpc.1
@@ -275,7 +275,7 @@ Tells the compiler to use pipes to commu
 Tells the compiler to include register allocation/deallocation information.
 .TP
 .BI \-at
-Tells the compiler to include temparary register allocation/deallocation information.
+Tells the compiler to include temporary register allocation/deallocation information.
 .TP
 .BI \-A xxx
 specifies what kind of assembler should be generated . Here
Index: fpc/install/man/man1/h2pas.1
===================================================================
--- fpc.orig/install/man/man1/h2pas.1
+++ fpc/install/man/man1/h2pas.1
@@ -100,7 +100,7 @@ defines are changed into pascal constant
 macros are changed - wherever possible to functions; however the arguments
 are all integers, so these must be changed manually. Simple expressions 
 in define staments are recognized, as are most arithmetic operators: 
-addition, substraction, multiplication, division, logical operators, 
+addition, subtraction, multiplication, division, logical operators,
 comparison operators, shift operators. The C construct ( A ? B : C)
 is also recognized and translated to a pascal construct with an IF
 statement (this is buggy, however).
